packages/devextreme-scss/images/icons/material/sparkle.svg

```diff
@@ -0,0 +1,4 @@
+<!-- Generated by IcoMoon.io -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+<path d="M19 9l1.25-2.75 2.75-1.25-2.75-1.25-1.25-2.75-1.25 2.75-2.75 1.25 2.75 1.25 1.25 2.75zM11.5 9.5l-2.5-5.5-2.5 5.5-5.5 2.5 5.5 2.5 2.5 5.5 2.5-5.5 5.5-2.5-5.5-2.5zM19 15l-1.25 2.75-2.75 1.25 2.75 1.25 1.25 2.75 1.25-2.75 2.75-1.25-2.75-1.25-1.25-2.75z"></path>
```

Comment:
Each icon should have title like: `<title>Sparkle</title>`
And remove comment, please. See existed .svg